documentation/modules/exploit/windows/http/fortilogger_arbitrary_fileupload.md

```diff
@@ -0,0 +1,192 @@
+## Vulnerable Application
+
+FortiLogger is a web-based logging and reporting software designed specifically for FortiGate firewalls,
+running on Windows operating systems. It contains features such as instant status tracking, logging, search / filtering,
+reporting and hotspot.
+
+This module exploits an unauthenticated arbitrary file upload via insecure `POST` request on company logo upload
+for hotspot settings of FortiLogger < 5.2.0.
+
+You can download installation files from https://www.fortilogger.com/download
+
+*Vendor has removed version 4.4.2.2 from web page of vendor for download.
+You can download version 4.4.2.2 from https://github.com/erberkan/erberkan.github.io/raw/master/2021/cve-2021-3378/Fortilogger-4.4.2.zip*
+
+Tested versions:
+
+- 4.4.2.2
+- 3.6.2.9
+- 3.4.1.7
+
+### Prerequisites
+
+1. Start a Windows VM *(Tested on Windows 10 Enterprise)*
+2. Install a vulnerable version which is any version less then 5.2.0 of FortiLogger via above url.
+3. After installation, verify that the server is working by visiting it with a browser.
+   - Default port: 5000
+   - Default username:password - admin:admin
+
+
+## Verification Steps
+
+1. Install the application
+1. Start msfconsole
+2. Do: `use exploit/windows/http/fortilogger_arbitrary_fileupload`
+3. Set `RHOSTS`
+4. Do: `run` or `exploit`
+5. **Verify** that `The target is vulnerable. FortiLogger version [version number]` message appeared
+6. **Verify** that payload uploaded to target system successfully: `Payload has been uploaded`
+7. **Verify** that you getting a meterpreter session.
+
+## Options
+
+1.  `RHOSTS`. IP address or FQDN of target system that is FortiLogger < 5.2.0 is running on it.
+2.  `RPORT`. Default port number is `5000`.
```

Comment:
Module options are supposed to use `###` headings.

There's a linting tool in `./dev/tools/msftidy_docs.rb` which you can use for linting module documentation. Although it didn't flag this for some reason.